tempesta_fw/http_parser.h

```diff
@@ -98,6 +131,8 @@ typedef struct {
 	TfwStr		_tmp_chunk;
 	TfwStr		hdr;
 	TfwHttpHbhHdrs	hbh_parser;
+	TfwHttpDateFmt	*date_fmt;
+	char		date_str[MAX_DATE_LEN];
```

Comment:
There is not reason to keep the temporal buffer - we don't appreciate Nginx-like copies and multiple data traversals.